### ISSUE TITLE
Curly bracket inside command causes 500 error.

### ISSUE THREAD

**Issue by sabinlinden** [tonybaloney/sphinxcontrib-confluencebuilder#41]
This line in a .rst:
`* Host is located at ``/var/lib/store/hosts/{hostname}``.
`

Caused this error:
```
sphinxcontrib.confluencebuilder error:
---
An unsupported Confluence API call has been made.

REQ: POST
RSP: 500
URL: https://company.atlassian.net/wiki/rest/api/
API: contentbody/convert/storage
---
```

Removing the curly brackets around hostname made the error go away.

**Comment by jdknight** [comment]
Related to #39.

**Comment by tonybaloney** [comment]
Fixed in #43  
![image](https://user-images.githubusercontent.com/1532417/29504281-ed242262-8681-11e7-92e1-100870a4f27e.png)

**Comment by tonybaloney** [comment]
@sabinlinden please can you validate this is fixed. 

`pip install -e git+https://github.com/tonybaloney/sphinxcontrib-confluencebuilder.git#egg=sphinxcontrib-confluencebuilder`

**Comment by sabinlinden** [comment]
Unfortunately the pip install command errors out in my current environment and I have yet to figure out the reason why. I'll let you know if I figure this out.

**Comment by spyoungtech** [comment]
@sabinlinden try upgrading setuptools. 

```
pip install --upgrade setuptools
```

**Comment by sabinlinden** [comment]
Sorry, that didn't work and I've been strapped for time so I still haven't resolved the issue with my environment.

**Comment by jdknight** [comment]
This issue should now be resolved in db5b916e57898eb86244b4c33a801f5da5c3b984 (available on the master; see specifically ca84f547796c091d80a131bc2edb8b5bfd435441 AND b6bb29230db22ccc45a5852c75f88ef4cfbea422). To take advantage of this change, one can pull from `master`. This change should be made available in an upcoming v0.7 release (#29).
